selfdrive/manager/process_config.py

```diff
@@ -35,8 +38,8 @@ def qcomgps(started, params, CP: car.CarParams) -> bool:
   NativeProcess("clocksd", "system/clocksd", ["./clocksd"]),
   NativeProcess("logcatd", "system/logcatd", ["./logcatd"]),
   NativeProcess("proclogd", "system/proclogd", ["./proclogd"]),
-  PythonProcess("logmessaged", "system.logmessaged", offroad=True),
-  PythonProcess("micd", "system.micd"),
+  PythonProcess("logmessaged", "system.logmessaged", offroad=True, callback=iscar),
```

Comment:
what's this for?

Reply:
I think this was to stuff the yolo message in this. We don't need this anymore.